src/reducers/operations.ts

```diff
@@ -100,6 +101,11 @@ export const operationsTracker: Middleware =
       // discard the `txEventContext` property since it's not serializable and irelevent.
       const {txEventContext: _,  ...options} = info.options;
 
+      let proposalTitle = options.title;
+      if (options.proposalId) {
+        proposalTitle = (getState() as any as IRootState).arc.proposals[options.proposalId].title;
```

Comment:
as any as IRootState?

Reply:
To make Typescript happy since it infers it to be a generic `S`. `getState: <S>() -> S`.